src/actionTypes/index.js

```diff
@@ -1,2 +1,4 @@
+import './signupActionTypes';
+
 export const EXAMPLE = 'EXAMPLE';
-export const EXAMPLETWO = 'EXAMPLE-TWO'
+export const EXAMPLE_TWO = 'EXAMPLE-TWO';
```

Comment:
Maintain consistency in type names and values. `const EXAMPLE_TWO = 'EXAMPLE_TWO';`

Reply:
I have fixed this.